openedx_events/management/commands/consume_events.py

```diff
@@ -0,0 +1,85 @@
+"""
+Makes ``consume_events`` management command available.
+"""
+import json
+import logging
+
+from django.core.management.base import BaseCommand
+
+from openedx_events.event_bus import make_single_consumer
+from openedx_events.tooling import OpenEdxPublicSignal, load_all_signals
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Management command for consumer workers in the event bus.
+    """
+
+    help = """
+    Consume messages of specified signal type from a topic and send their data to that signal.
+
+    Example::
+
+        python3 manage.py consume_events -t user-login -g user-activity-service \
+            -s org.openedx.learning.auth.session.login.completed.v1
+
+        # send extra args, for example pass check_backlog flag to redis consumer
```

Comment:
can we not be technology specific? or is it needed?

Reply:
@mariajgrimaldi there are additional features that we can use depending on the underlying tech, for example kafka has something related to resetting offset (not exactly sure how it works) and redis streams allows to start consuming from a specific id or read all the consumer groups unread msgs, claim messages from other consumers etc. The default behavior is same i.e. read new msgs only.

Reply:
I get it now! Thank you